[Bug]: beam_PostCommit_Java permared DLPTextOperationsIT.inspectsText failing

### What happened?

Error message:

```
Stacktrace
java.lang.AssertionError: DLPInspectText/DLPInspect/ParMultiDo(InspectData).output: null
	at org.apache.beam.sdk.testing.PAssert$PAssertionSite.capture(PAssert.java:175)
	at org.apache.beam.sdk.testing.PAssert.that(PAssert.java:460)
	at org.apache.beam.sdk.testing.PAssert.that(PAssert.java:452)
	at org.apache.beam.sdk.extensions.ml.DLPTextOperationsIT.inspectsText(DLPTextOperationsIT.java:71)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        ...
Caused by: java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at org.apache.beam.sdk.extensions.ml.DLPTextOperationsIT$VerifyInspectionResult.apply(DLPTextOperationsIT.java:121)
	at org.apache.beam.sdk.extensions.ml.DLPTextOperationsIT$VerifyInspectionResult.apply(DLPTextOperationsIT.java:109)
        ....
org.apache.beam.repackaged.direct_java.runners.core.SimplePushbackSideInputDoFnRunner.processElementInReadyWindows(SimplePushbackSideInputDoFnRunner.java:79)
	at org.apache.beam.runners.direct.ParDoEvaluator.processElement(ParDoEvaluator.java:244)
	at org.apache.beam.runners.direct.DoFnLifecycleManagerRemovingTransformEvaluator.processElement(DoFnLifecycleManagerRemovingTransformEvaluator.java:54)
	at org.apache.beam.runners.direct.DirectTransformExecutor.processElements(DirectTransformExecutor.java:165)
	at org.apache.beam.runners.direct.DirectTransformExecutor.run(DirectTransformExecutor.java:129)
```

It start failing on May 4, 5 PM GMT, but then at May 5, 5 PM GMT there is a successful run, then permared since then. No obvious code change at that time. Is it some breaking change at gcp side?

<img width="346" alt="image" src="https://user-images.githubusercontent.com/8010435/236847651-1ead4732-a3bd-489a-ab8b-d03ddf405284.png">

### Issue Priority

Priority: 1 (data loss / total loss of function)

### Issue Components

- [ ] Component: Python SDK
- [X] Component: Java SDK
- [ ] Component: Go SDK
- [ ] Component: Typescript SDK
- [ ] Component: IO connector
- [ ] Component: Beam examples
- [ ] Component: Beam playground
- [ ] Component: Beam katas
- [ ] Component: Website
- [ ] Component: Spark Runner
- [ ] Component: Flink Runner
- [ ] Component: Samza Runner
- [ ] Component: Twister2 Runner
- [ ] Component: Hazelcast Jet Runner
- [ ] Component: Google Cloud Dataflow Runner

--- a/sdks/java/extensions/ml/src/test/java/org/apache/beam/sdk/extensions/ml/DLPTextOperationsIT.java
+++ b/sdks/java/extensions/ml/src/test/java/org/apache/beam/sdk/extensions/ml/DLPTextOperationsIT.java
@@ -116,7 +116,8 @@ public class DLPTextOperationsIT {
             List<Finding> resultList = item.getValue().getResult().getFindingsList();
             matches.add(
                 resultList.stream()
-                    .anyMatch(finding -> finding.getInfoType().equals(emailAddress)));
+                    .anyMatch(
+                        finding -> finding.getInfoType().getName().equals(emailAddress.getName())));
           });
       assertTrue(matches.contains(Boolean.TRUE));
       return null;

